addons/product_expiry/models/stock_move_line.py

```diff
@@ -9,7 +9,8 @@
 class StockMoveLine(models.Model):
     _inherit = "stock.move.line"
 
-    expiration_date = fields.Datetime(string='Expiration Date', compute='_compute_expiration_date', store=True,
+    expiration_date = fields.Datetime(
```

Comment:
#UselessDiff

Reply:
But it causes a linter error :crying_cat_face:

Reply:
![](https://media4.giphy.com/media/LfwMyRpf9X6q4/giphy.gif)